public/images/user.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="14" height="16" viewBox="0 0 14 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this and the following image `public/images/wikidata-logo.svg` are generated, right? So I guess they shouldn't be added to git.

Reply:
Addressed!